rapids/internal/install-gpu-driver.sh

```diff
@@ -0,0 +1,13 @@
+#!/bin/bash
+
+apt-get update
+apt-get install -y pciutils
+export DEBIAN_FRONTEND=noninteractive
+apt-get install -y "linux-headers-$(uname -r)"
+
+# TODO: Detect attached GPU type, autopick recent driver URL
```

Comment:
I'm not sure if there is an easy way of doing this in the VM. `lspci` might help.

Reply:
I confirmed that for all GPU types RAPIDS works with in Dataproc (everything but K80s), the same driver URL works. Removing this comment.

For future reference, re: lspci, I don't believe it gives you enough information to determine card type until the driver is installed.